contrib/miniupnpd/files/miniupnpd

```diff
@@ -0,0 +1,6 @@
+# miniupnpd service
+
+type = process
+command = /usr/bin/miniupnpd -f /etc/miniupnpd/miniupnpd.conf
+depends-on = pre-network.target
+log-type = buffer
```

Comment:
how much does it log? because buffer logtype only captures enough to fill the buffer and does not scroll

Reply:
well you'd probably want logfile= then

also i think you forgot to commit the changes for these

Reply:
buffer will be enough for it to throw possible error messages